lib/fastlane/actions/slather.rb

```diff
@@ -6,9 +6,16 @@ module SharedValues
 
     class SlatherAction < Action
       def self.run(params)
-        Actions.verify_gem!('slather')
+        #This will fail if using Bundler. Skip the check rather than needing to
+        # require bundler
+        use_bundler = params[:use_bundle_exec]
+        if ! use_bundler
```

Comment:
Favor `unless` over `if` for negative conditions.<br>Do not leave space between `!` and its argument.